modules/top-level/nixpkgs.nix

```diff
@@ -0,0 +1,78 @@
+{
+  config,
+  options,
+  lib,
+  ...
+}:
+let
+  cfg = config.nixpkgs;
+  opt = options.nixpkgs;
+in
+{
+  options.nixpkgs = {
+    pkgs = lib.mkOption {
+      # TODO:
+      # defaultText = lib.literalExpression ''
+      #   import "''${nixos}/.." {
+      #     inherit (cfg) config overlays localSystem crossSystem;
+      #   }
+      # '';
+      type = lib.types.pkgs // {
+        description = "An evaluation of Nixpkgs; the top level attribute set of packages";
+      };
+      example = lib.literalExpression "import <nixpkgs> { }";
+      description = ''
+        If set, the `pkgs` argument to all Nixvim modules is the value of this option.
+
+        <!-- TODO: remove -->
```

Comment:
Forgot to ask what was going on with the multiple TODO's here.

Reply:
Each TODO relates to a section of documentation that should be added/removed once the other `nixpkgs.*` options are implemented.

Reply:
It currently renders as:

![image](https://github.com/user-attachments/assets/c0863f8a-1c11-47cb-be83-837c975b4c95)